modules/ROOT/pages/query/operations.adoc

```diff
@@ -114,16 +114,17 @@ Parameters can be used for:
 
 * literals and expressions
 * node and relationship IDs
```

Comment:
Just curious why this gets called out specifically? Is it also called out on docs elsewhere?

Reply:
It was raised as an issue on our wall and it reflects what's in the Cypher manual: https://neo4j.com/docs/cypher-manual/current/syntax/parameters/#cypher-parameters-introduction